hls4ml/model/optimizer/passes/bipolar_quant_opt.py

```diff
@@ -0,0 +1,250 @@
+"""
```

Comment:
The stuff in here are quite significant overlap with the normal quant_opt.py. I am wondering if it would be better to merge the two and handle the bipolar stuff as as special cases in the same optimizers. But that would result in a bunch of if statements, so I'm not sure which option would be cleaner and easier to maintain.

Reply:
This was a design choice originally made by Jure, but I think that there are enough differences in the behavior that it's cleaner to have them as separate optimizers. It's not a strong opinion, though.